apps/src/feedback.js

```diff
@@ -462,16 +465,14 @@ FeedbackUtils.prototype.displayFeedback = function (options, requiredBlocks,
    * @param {boolean} shouldPublish Whether to publish. Default: false.
    * @param {function} callback Callback function to call on success.
    */
-  function addSaveProjectButtonHandler(buttonSelector, pendingText, finishedText, shouldPublish, callback) {
+  function addSaveProjectButtonHandler(buttonSelector, pendingText, shouldPublish, callback) {
     const buttonElement = feedback.querySelector(buttonSelector);
     if (buttonElement) {
       dom.addClickTouchEvent(buttonElement, () => {
         $(buttonSelector).prop('disabled', true).text(pendingText);
-        project.copy(project.getNewProjectName(), () => {
-          FeedbackUtils.saveThumbnail(options.feedbackImage).then(() => {
-            $(buttonSelector).prop('disabled', true).text(finishedText);
-          }).then(callback);
-        }, {shouldPublish});
+        project.copy(project.getNewProjectName(), {shouldPublish})
+          .then(() => FeedbackUtils.saveThumbnail(options.feedbackImage))
+          .then(callback);
```

Comment:
Another unhandled possible rejection, though you didn't introduce this one and I wouldn't consider it blocking to fix this.